tests/e2e/link_test.go

```diff
@@ -63,6 +63,13 @@ var _ = Describe("odoLinkE2e", func() {
 			})
 		})
 
+		It("app describe should show the mysql service", func() {
```

Comment:
LGTM. UTs won’t be a good idea here. Only user facing scenario (Integration test) would be enough jot verify a description change. 

However i want to test the pr just to see the output format